### PR TITLE
Add cert expiration length flag

### PR DESCRIPTION
**Pull request by bronzdoc**
# Description:
This PR adds the ability to specify an expiration length with the gem cert command using the `-d/--days` flag

closes https://github.com/rubygems/rubygems/issues/1719

cc @djberg96 
______________

# Tasks:

- [ ] Describe the problem / feature
- [ ] Write tests
- [ ] Write code to solve the problem
- [ ] Get code review from coworkers / friends

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: lib/rubygems/security.rb [hunk 1]
```diff
@@ -377,6 +377,11 @@ class Exception < Gem::Exception; end
   ONE_YEAR = 86400 * 365
 
   ##
+  # One day in seconds
+
+  ONE_DAY = 86400
```
Comment (djberg96) [inline lib/rubygems/security.rb]: This is fine, but then I would say we could move ONE_YEAR below it and change it to:

    ONE_YEAR = ONE_DAY * 365

File: lib/rubygems/commands/cert_command.rb [hunk 2]
```diff
@@ -84,6 +84,11 @@ def initialize
 
       options[:sign] << cert_file
     end
+
+    add_option('-d', '--days NUMBER_OF_DAYS',
+               'Days for the certificate to expire') do |days, options|
```
Comment (djberg96) [inline lib/rubygems/commands/cert_command.rb]: This phrase sounds a bit odd. How about "Days before the certificate expires".

**Comment by djberg96** [comment]
A couple very minor nits that I've added comments on, but overall great! Thanks!

File: lib/rubygems/security.rb (hunk 1 above)
Comment (bronzdoc) [inline lib/rubygems/security.rb]: :+1:

File: lib/rubygems/commands/cert_command.rb (hunk 2 above)
Comment (bronzdoc) [inline lib/rubygems/commands/cert_command.rb]: Yeah that sounds better, thanks.

**Comment by bronzdoc** [comment]
Changes done @djberg96

**Comment by djberg96** [comment]
👍 LGTM

**Comment by bronzdoc** [comment]
@homu r+

**Pull request merged**